resin/models/device.py

```diff
@@ -60,10 +61,13 @@ def get(self, uuid):
             'eq': uuid
         }
         try:
-            return self.base_request.request(
+            devices = self.base_request.request(
                 'device', 'GET', params=params,
                 endpoint=self.settings.get('pine_endpoint')
-            )['d'][0]
+            )['d']
+            if len(devices) > 1:
```

Comment:
The same as application, it's added in JS SDK https://github.com/resin-io/resin-sdk/blob/master/lib/models/device.coffee#L293-L294